src/uk/gov/hmcts/contino/AngularBuilder.groovy

```diff
@@ -15,6 +15,11 @@ class AngularBuilder extends AbstractBuilder {
     builder.yarn("build:ssr")
   }
 
+  @Override
```

Comment:
the code in the PR looks fine, a couple of questions though:

1. Why create a java client rather than a bash script, it looks like some files need zipping and then a java jar invoked?
2. Why use `Runtime` to call a java jar on the command line, wouldn't it be less effort to add the jar to the classpath and call the code directly? rather than messing with process args, and string concatentation

Reply:
Thank you @timja. Answers are as below:

1- There is and will be a significant deal of further logic that would be much nicer and better maintainable in java. (Simple html report generation, properties management, Jira integration with rest-apis, etc) Obviously, all those can still be implemented as a script, but much more convenient in a strongly typed, OOP way with enablement of huge deals of open source provisions for any later needs.

2- I had asked the same to Kaan, and he clarified that the code in the jar is written to check a particular jar version from which it is being executed, hence the work around. Otherwise, we'd actually call an object interface within the JVM. However, our prospect is that the original fortify client jar we are wrapping will later on be published in a more convenient way, along with more open support for code re-use, and then the hmcts' fortify client wrapper can be refactored seamlessly.